tidb-lightning/tidb-lightning-physical-import-mode.md

```diff
@@ -74,7 +74,7 @@ It is recommended that you allocate CPU more than 32 cores and memory greater th
 
 - Do not use the physical import mode to directly import data to TiDB clusters in production. It has severe performance implications. If you need to do so, refer to [Pause scheduling on the table level](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#scope-of-pausing-scheduling-during-import).
 - If your TiDB cluster has a latency-sensitive application and a low concurrency, it is strongly recommended that you **do not** use the physical import mode to import data into the cluster. This mode might have significant impact on the online application.
-- Do not use multiple TiDB Lightning instances to import data to the same TiDB cluster by default. Use [Parallel Import](/tidb-lightning/tidb-lightning-distributed-import.md) instead.
+- If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, enable [parallel import](/tidb-lightning/tidb-lightning-distributed-import.md).
```

Comment:
```suggestion
- If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, better enable [parallel import](/tidb-lightning/tidb-lightning-distributed-import.md).
```
We actually support running multiple TiDB Lightning instances without `parallel import` enabled, but from the original context, it seems a suggestion